src/Agent/NewRelic/Agent/Core/DataTransport/ConnectionManager.cs

```diff
@@ -107,12 +109,14 @@ private void Connect()
             {
                 HandleHttpErrorResponse(ex);
             }
+#if !NETFRAMEWORK // Only available in System.Net.Http
```

Comment:
What is the fallback behavior if we are instrumenting a .NET Framework app?  Is there a different exception type thrown that is caught with similar behavior (`ScheduleRestart()`), or does it go to the generic exception catch which causes the agent to disconnect?

Reply:
That's a good question. As far as I can tell there's no equivalent, so it will fall through to the shutdown case. This won't change that behavior, though, since it wasn't possible to generate that exception.

Reply:
Good point. 👍